service/controller/resource/deployment/template/main.json

```diff
@@ -810,56 +810,7 @@
                   "addressPrefixes":[
                     "[parameters('virtualNetworkCidr')]"
                   ]
-                },
-                "subnets":[
-                  {
-                    "name":"[variables('masterSubnetName')]",
-                    "properties":{
-                      "addressPrefix":"[parameters('masterSubnetCidr')]",
-                      "networkSecurityGroup":{
-                        "id":"[parameters('masterSecurityGroupID')]"
-                      },
-                      "routeTable":{
-                        "id":"[parameters('routeTableID')]"
-                      },
-                      "serviceEndpoints": [
-                        { "service": "Microsoft.Storage" }
-                      ]
-                    }
-                  },
-                  {
-                    "name":"[variables('workerSubnetName')]",
-                    "properties":{
-                      "addressPrefix":"[parameters('workerSubnetCidr')]",
-                      "natGateway": {
-                        "id":"[parameters('workersNatGWID')]"
-                      },
-                      "networkSecurityGroup":{
-                        "id":"[parameters('workerSecurityGroupID')]"
-                      },
-                      "routeTable":{
-                        "id":"[parameters('routeTableID')]"
-                      },
-                      "serviceEndpoints": [
-                        { "service": "Microsoft.Storage" },
-                        { "service": "Microsoft.Sql" },
-                        { "service": "Microsoft.AzureCosmosDB" },
-                        { "service": "Microsoft.KeyVault" },
-                        { "service": "Microsoft.ServiceBus" },
-                        { "service": "Microsoft.EventHub" },
-                        { "service": "Microsoft.AzureActiveDirectory" },
-                        { "service": "Microsoft.ContainerRegistry" },
-                        { "service": "Microsoft.Web" }
-                      ]
-                    }
-                  },
-                  {
-                    "name":"[parameters('vnetGatewaySubnetName')]",
-                    "properties":{
-                      "addressPrefix":"[parameters('vpnSubnetCidr')]"
-                    }
-                  }
-                ]
+                }
```

Comment:
We no longer create the subnets as part of the virtual network definition, but as separate deployments.